arch/arm/boot/dts/overlays/i2c-fan-overlay.dts

```diff
@@ -83,8 +78,8 @@
 		i2c_csi_dsi =	<&frag100>,"target:0=",<&i2c_csi_dsi>,
 				<0>,"+101+102";
 		addr =		<&emc2301>,"reg:0";
-		minrpm =	<&emc2301>,"min-rpm;0";
-		maxrpm =	<&emc2301>,"max-rpm;0";
+		minrpm =	<&emc2301>,"emc2305,pwm-min;0";
```

Comment:
The documentation uses `minpwm` and `maxpwm`, which make more sense. But it highlights that the PWM setting is device-specific in a way that an RPM setting wouldn't be (provided the driver supports RPMs). Perhaps this would be better as a dedicated CM4 fan overlay? I don't know.

Reply:
> The documentation uses minpwm and maxpwm, which make more sense

Yes replaced the docs but not the override name.

This is the downside of the emc2305 driver over the emc2301 one - PWM setting vs RPM.

Then again the RPM range is variable between any two fans. At least default PWM settings should guarantee the fan spins (the controller does give a boost if it thinks it is stalled), whereas the default speed settings won't. (eg my fan here has a range of 200 to 1800 rpm, so the defaults you had used of 3500 and 5500 rpm aren't applicable)